parasl/parasl.txt

```diff
@@ -51,12 +56,20 @@ v0 : int(16) = 0; // 16-bit int
 
 Default int is 32.
 
-Integer are two's complement signed numbers with two's complement signed wrap. No UB here.
+Integer are two's complement signed numbers with two's complement signed wrap.
+No UB here.
+
+char type is something like std::byte in C++.
 
 1.3. Arrays
 
-Array entity is syntactic glue for the couple of entities of the same type. Array have CT-known size.
-Special arryas are vectors. Vectors have both CT-known size and CT-known indexes.
+Array entity is syntactic glue for the couple of entities of the same type.
+Array is something like VLA in C. It locates on stack and can have CT-known
```

Comment:
Array of not CT-known size is not VLA. If size is compile time known, it is fixed. VLA are only CT-unknown-sized arrays.